src/System.Private.ServiceModel/tools/test/Bridge/Bridge/ResourceController.cs

```diff
@@ -182,30 +195,39 @@ public HttpResponseMessage Delete(HttpRequestMessage request)
             }
         }
 
+        private HttpResponseMessage BuildJsonContent(string contentString)
+        {
+            // Directly return a json string to avoid use of MediaTypeFormatters  
+            HttpResponseMessage response = Request.CreateResponse(HttpStatusCode.OK);
+            response.Content = new StringContent(contentString);
+            response.Content.Headers.ContentType = new MediaTypeHeaderValue(JsonSerializer.JsonMediaType);
+            return response;
+        }
 
-        private Dictionary<string, string> BuildProperites(string resourceName)
+        private Dictionary<string, string> BuildProperties(string resourceName)
         {
             HttpRequestMessage request = this.Request;
 
-            // GET allows name/value pairs in Uri query parameters only
+            // GET allows name/value pairs in Uri query parameters only  
             Dictionary<string, string> properties = GetNameValuePairsFromQueryParameters(request);
 
-            if (!properties.ContainsKey("name"))
+            // PUT allows name/value pairs in Uri query parameters or in content.  
+            // Give precedence to query parameters.  
+            // If there were no query parameters, allow the content to provide it.  
+            if (properties.Count == 0 && request.Content != null)
             {
-                properties["name"] = resourceName;
+                properties = GetNameValuePairsFromContent(request);
             }
 
-            // PUT allows name/value pairs in Uri query parameters or in content.
-            // Give precedence to query parameters.
-            // If there were no query parameters, allow the content to provide it.
-            if (properties.Count == 0 && request.Content != null)
+            if (!properties.ContainsKey("name"))
             {
-                properties = GetNameValuePairsFromContent(request);
+                properties["name"] = resourceName;
             }
```

Comment:
Added a const to ResourceController.

We had this keyName for another resource, but we haven't made the set of keyname consts shared across all Controllers and IResources yet.